.circleci/config.yml

```diff
@@ -569,76 +570,39 @@ jobs:
       - run:
           name: Archive Unpacked Directories
           command: |
-            tar -zcf desktop/release/win-unpacked-x64.tar.gz desktop/release/win-unpacked
+            tar -zcf desktop/Automattic/wp-desktop/release/win-unpacked-x64.tar.gz desktop/Automattic/wp-desktop/release/win-unpacked
 
             # Check if win-ia32 build is present before archiving. If this is a full
             # artifact build, the win-ia32-unpacked executable will be generated and we can archive it.
-            If ( $(Test-Path -Path desktop\release\win-ia32-unpacked) ) {
-              tar -zcf desktop/release/win-unpacked-ia32.tar.gz desktop/release/win-ia32-unpacked
+            If ( $(Test-Path -Path desktop\Automattic\wp-desktop\release\win-ia32-unpacked) ) {
+              tar -zcf desktop/Automattic/wp-desktop/release/win-unpacked-ia32.tar.gz desktop/Automattic/wp-desktop/release/win-ia32-unpacked
             } else {
-              echo "Skipping tar archive for 'desktop\release\win-ia32-unpacked' (not built)"
+              echo "Skipping tar archive for 'desktop\Automattic\wp-desktop\release\win-ia32-unpacked' (not built)"
             }
       - run:
           when: always
           name: Clean Up
           command: |
             set +e
-            rm -rf desktop/release/github
-            rm -rf desktop/release/win-unpacked
-            rm -rf desktop/release/win-ia32-unpacked
+            rm -rf desktop/Automattic/wp-desktop/release/github
+            rm -rf desktop/Automattic/wp-desktop/release/win-unpacked
+            rm -rf desktop/Automattic/wp-desktop/release/win-ia32-unpacked
           shell: bash.exe
       - store_artifacts:
           when: always
-          path: desktop\release
+          path: desktop\Automattic\wp-desktop\release
       - persist_to_workspace:
           root: C:\Users\circleci\wp-calypso
           paths:
-            - desktop\release
-
-  # Publish all the artifacts generated by the previous wp-desktop- jobs to a
-  # GitHub release and generate the release notes using the Git commits history
-  wp-desktop-publish:
-    docker:
-      - image: circleci/golang:1.12
-    working_directory: /home/circleci/wp-calypso
-    environment:
-      VERSION: << pipeline.git.tag >>
-    steps:
-      - checkout
-      - attach_workspace:
-          at: /home/circleci/wp-calypso
-      - run:
-          name: Install Dependencies
-          command: go get github.com/tcnksm/ghr
-      - run:
-          name: Publish Github Release
-          command: |
-            echo "Publishing draft release for wp-desktop $VERSION..."
-            NAME="WP-Desktop ${VERSION#?}"
-
-            chmod +x desktop/bin/make-changelog.sh
-            ./desktop/bin/make-changelog.sh > desktop/CHANGELOG.md
-
-            ghr \
-              --token "${GH_TOKEN}" \
-              --username "${CIRCLE_PROJECT_USERNAME}" \
-              --repository "${CIRCLE_PROJECT_REPONAME}" \
```

Comment:
Looks like `ghr` can publish to _any_ GitHub repo. 
Maybe we can replace this entire PR with passing `--repository "Automattic/wp-desktop"` here?

Reply:
Update: I did a test with a fresh new repo and it seems to work.

```
$ mkdir temp2
$ echo test >> temp2/hello.txt
$ GITHUB_TOKEN=... ghr -u scinos -r 'test-ghr' v0.1.0 temp2
==> Create a new release
--> Uploading:       hello.txt
```

And it creates https://github.com/scinos/test-ghr/releases/tag/v0.2.0, which is tagged as `0.2.0` and has `hello.txt` attached.
```